src/commonTest/kotlin/it/krzeminski/snakeyaml/engine/kmp/CommonTestResourcesAccess.kt

```diff
@@ -33,12 +33,22 @@ fun stringFromResources(path: String): String {
 /**
  * Exposes files stored in `commonTest/resources` as a [FileSystem].
  */
-object CommonTestResourcesFileSystem : FileSystem() {
+val CommonTestResourcesFileSystem = buildFileSystem(CommonTestResources.resourcesMap)
```

Comment:
`CommonTestResourcesFileSystem` becomes just one (so far) of possible file systems created using the generated `resourceMap`.